runtime/src/chpl-cache-support.c

```diff
@@ -56,33 +57,6 @@ static inline void saturating_add(saturating_count_t *x, unsigned int num) {
   *x = v;
 }
 */
-// If we have a mask representing 2^n - 1,
```

Comment:
Earlier versions of the branch used round_up_to_mask, so I moved that function to a different header. I could remove it from the current version and possibly do it in a separate PR.